plugins/services/src/js/components/modals/ServiceModals.js

```diff
@@ -39,6 +40,19 @@ class ServiceModals extends React.Component {
     );
   }
 
+  getCannotDestroyGroupModal(groupFrameworks) {
+    const { onClose, modalProps } = this.props;
+    const key = ActionKeys.GROUP_DELETE;
+
+    return (
+      <GroupContainsFrameworksModal
+        services={groupFrameworks}
```

Comment:
Looks like we don't use the `services` prop in this component so let's remove.

Reply:
removed